playbot/playbot.py

```diff
@@ -25,6 +25,8 @@ def __init__(self, host, port):
         self.moves = ['NOP()', 'PATCH()', 'SCAN()', 'OVERLOAD()', 'OVERHEAR()', 'EXPLOIT()', 'INFECT()']
         self.host = host
         self.port = port
+        self.my_move = ''
+        self.move_ok = False
         self.socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         self.game = True
```

Comment:
This is the main variable that determines whether the game works.
I think it is better to init it with False and set to True if connection, login and other nessesery things go right. 
Just look how many times you have to set it to False if something fail.

Reply:
make sense - `self.game` will be initialized as `False`